packages/unit/tsconfig.json

```diff
@@ -1,3 +1,14 @@
 {
-  "extends": "../../tsconfig.json"
-}
\ No newline at end of file
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "types": ["node", "jest", "vitest", "@types/sinon"],
```

Comment:
> Adding those types here seems problematic since the unit package isn't linked to any dependency injection or mocking library adapter (e.g. @suites/types.lib). Is there a specific reason for including them?

Also here